Cargo.toml

```diff
@@ -4,29 +4,49 @@ description = "a proxy to access Momento cache(s) over memcache protocol"
 authors = ["Brian Martin <brayniac@gmail.com>"]
 
 version = "0.3.2"
-edition = { workspace = true }
-homepage = { workspace = true }
-repository = { workspace = true }
-license = { workspace = true }
+edition = "2021"
+homepage = "https://pelikan.io"
+repository = "https://github.com/pelikan-io/pelikan"
+license = "Apache-2.0"
 
 [dependencies]
-backtrace = { workspace = true }
-bytes = { workspace = true }
-clap = { workspace = true }
-clocksource = { workspace = true }
-common = { path = "../../common" }
-config = { path = "../../config" }
-crossbeam-channel = { workspace = true }
-futures = { workspace = true }
-libc = { workspace = true }
-logger = { path = "../../logger" }
-metriken = { workspace = true }
+backtrace = "0.3.69"
+bytes = "1.7.2"
+clap = "4.4.6"
+clocksource = "0.8.1"
+common = { git = "https://github.com/pelikan-io/pelikan.git", branch = "main", package = "common" }
+config = { git = "https://github.com/pelikan-io/pelikan.git", branch = "main", package = "config" }
```

Comment:
For the various git references, can we pin to a commit? Otherwise this is a moving target.